### ISSUE TITLE
Discarding doesn't always match

### ISSUE THREAD

**Issue by jjwilbur** [thyer/CS340winter2016#223]
When a 7 is rolled the amount in your hand does not always match how many it has as max value to discard.  For Example I had 17 wood in my hand and the discard window wanted me to discard 10 cards and said that I had 20.

**Comment by jakebrucebrewer** [comment]
@jjwilbur Could you show us documentation of this? It would be helpful since we've never seen this happen. What were the other circumstances involved?

**Comment by jpp46** [comment]
@jjwilbur @thyer 
I haven't been able too duplicate this bug ever, no matter how many games I play and no matter what number of clients I use. I cannot see any reason for it to happen in the java code either. I am not sure what to do here? More info or a screenshot here would have been helpful, or the steps leading up to it.

**Comment by jjwilbur** [comment]
@jpp46 Sorry, I wish I had documented it better.  I'm not exactly sure how I got it to happen. It could very well have been after the game had finished and I was still playing when it happened.  All I remember was that I had 17 wood in my hand and when the discard window popped up it was asking me to discard 10 and said that I had 20 to discard in the discard window. 
This very well could be invalid because the game might have been over when I came across this. I know the game does weird things if it is was finished.
